policyengine_us/parameters/gov/states/hi/tax/income/exemptions/base.yaml

```
@@ -0,0 +1,16 @@
+description: Hawaii provides this base exemption amount.
+values:
+  2021-01-01: 1_144
```

Can you find how it's adjusted? This would be ~$3,000 today if inflation-adjusted, and the disability exemption hasn't changed.
```suggestion
  # Legal code specifies $1,040 as of 1985.
  2021-01-01: 1_144
```

---

it seemed to have increased per the tax form but we cannot find a bill that would support that

---

ok please file an issue to investigate

---

[New issue ](https://github.com/PolicyEngine/policyengine-us/issues/3243)